### PR TITLE
feat(text-input): provide ecl-text-input class - INNO-520

### PR DESCRIPTION
**Pull request by yhuard**
Reference: https://ec-europa.github.io/ec-europa-theme/section-form.html#kssref-form-text-input

TODO: 
- [x] Provide tests

### CODE REVIEW

File: framework/components/ecl-forms/ecl-forms-form-groups/_form-groups.scss [hunk 1]
```diff
@@ -1,12 +1,9 @@
-/*
-Form Group
+/**
+ * Form Group
+ * @define form-group
+ */
 
-Markup: html/form-group.html
```
Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-form-groups/_form-groups.scss]: The `ecl-forms-form-groups.twig` still has old classes.

File: framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
   title: 'Text inputs',
   label: 'Text inputs',
-  status: 'planned',
+  status: 'ready',
```
Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.config.js]: The `help-block` part is still in progress though.

File: framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.twig [hunk 3]
```diff
@@ -1,29 +1,21 @@
 <form>
-  <div class="form-group">
+  <div class="ecl-form-group">
     <label for="example-input-id-1">Text Input - default with placeholder</label>
     <p class="help-block">This is some help text.</p>
-    <input type="text" class="form-control" id="example-input-id-1" placeholder="Some placeholder text."/>
+    <input type="text" class="ecl-text-input" id="example-input-id-1" placeholder="Some placeholder text."/>
```
Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.twig]: If possible, please order the ids and names.
Also, it seems that some examples are missing: https://ec-europa.github.io/ec-europa-theme/section-form.html#kssref-form-text-input

File: framework/components/ecl-forms/ecl-forms-text-inputs/package.json [hunk 4]
```diff
@@ -1,8 +1,20 @@
 {
-  "private": true,
   "name": "@ec-europa/ecl-forms-text-inputs",
```
Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-text-inputs/package.json]: Either please add `"@ec-europa/ecl-forms-help-blocks"` as a dependency or maybe just remove it for the time being.

File: framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.twig (hunk 3 above)
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.twig]: No missing examples actually: they made an example for the `:focus` state by adding a class for that... Do we want to follow this road?

File: framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.config.js (hunk 2 above)
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-text-inputs/ecl-forms-text-inputs.config.js]: I'll rework this

File: framework/components/ecl-forms/ecl-forms-text-inputs/package.json (hunk 4 above)
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-text-inputs/package.json]: I'll add it as a peerDependency, thanks

Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-text-inputs/package.json]: Should be better now

**Comment by yhuard** [comment]
Ready for a last review :)

**Pull request merged**